src/EVM/Types.hs

```diff
@@ -1006,11 +1008,14 @@ data GenericOp a
   | OpSelfbalance
   | OpBaseFee
   | OpPop
+  | OpMcopy
   | OpMload
   | OpMstore
   | OpMstore8
   | OpSload
   | OpSstore
+  | OpTLoad
```

Comment:
If we follow the other ops style this should be `OpTload` like in `OpSload`

Reply:
Oh, good point. Fixing.